distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -279,12 +280,39 @@ public void testReadDerivedStorage() throws IOException {
   }
 
   @Test
-  public void testMetacardTransformersFromRest() {
-    String id = ingestGeoJson(getFileContent(JSON_RECORD_RESOURCE_PATH + "/SimpleGeoJsonRecord"));
+  public void testGeoJsonMetacardTransformersFromRest() {
+    String id =
+        ingestGeoJson(
+            getFileContent(JSON_RECORD_RESOURCE_PATH + "/SimpleGeoJsonWithMetadataRecord"));
+    String title = "myTitle";
+    String geoCoordinates = "30.0 10.0";
+
+    String idXPath = "/metacard[@id='" + id + "']";
+    String titleXPath = "/metacard/string[@name='title'][value='" + title + "']";
+    String geoXPath = "/metacard/geometry";
+    String geoCoordsXPath = geoXPath + "//Point[pos='" + geoCoordinates + "']";
+    String metadataXPath = "/metacard/stringxml[@name='metadata']/value/metacard";
+    String metadataSourceXPath = metadataXPath + "[source='ddf.distribution']";
+    String metadataExtractedTextXPath =
+        metadataXPath + "/string[@name='ext.extracted.text'][value='sample extracted text']";
 
     String url = REST_PATH.getUrl() + id;
     LOGGER.info("Getting response to {}", url);
-    when().get(url).then().log().all().assertThat().body(hasXPath("/metacard[@id='" + id + "']"));
+
+    // tests all of the XPath to verify that metacard properties are properly set
```

Comment:
Might want to elaborate here? I think you're saying that the metadata attributes are parsed and mixed with the json properties.

Reply:
Yeah, I just changed it. How does the new comment sound?

Reply:
Looks good to me. 👍